org.srb2.SRB2.metainfo.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2020 Flathub maintainers -->
+<component type="desktop-application">
+  <id>org.srb2.SRB2</id>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <name>Sonic Robo Blast 2</name>
+  <summary>A 3D Sonic the Hedgehog fangame based on a modified version of Doom Legacy</summary>
+  <description>
```

Comment:
Please mention that it requires external assets to work.

Reply:
Done